proposals/kubeflow-steering-committee-charter.md

```diff
@@ -0,0 +1,118 @@
+-  Trademark guidance is delegated to the Conformance Committee.
+
+## Committee Meetings
+
+KSC currently meets **<frequency TBD>**, or as-needed. Meetings are open to the public and held online.
+
+Given the private nature of some of these discussions (e.g. privacy, private emails to the committee, code of conduct violations, escalations, disputes between members, security reports, etc.), some meetings are held in private.
+
+Meeting notes are available to members of the kubeflow-discuss mailing list, unless community member privacy requires otherwise. Public meetings will be recorded and the recordings made available publicly.
+
+Questions and proposals for changes to governance are posted as issues in the kubeflow/community repo, and the KSC invites your feedback there. See [Getting in touch](#getting-in-touch) for other options.
+
+## Committee members
+
+KSC is composed of 5 (five) members. They are elected according to the election policy [TODO: add link].  
+Seats on the Steering Committee are held by an individual, not by their employer.  
+The current membership of the committee is currently (listed alphabetically by first name):
+
+<table>
+  <thead>
+    <tr>
+      <th><br>
+Member</th>
+      <th><br>
+Organization</th>
+      <th><br>
+Github Profile</th>
+      <th><br>
+Term Start</th>
+      <th><br>
+Term End</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+    <tr>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+      <td></td>
+    </tr>
+  </tbody>
+</table>
+
+## Emeritus Committee Members
+
+[This section will be populated when there are retired committee members.]
+
+## Decision process
+
+The steering committee desires to always reach consensus.
+
+Decisions requiring a vote include: issuing written policy, amending existing written policy, creating, removing, or modifying a working group; all spending, hiring, and contracting decisions; official responses to publicly raised issues; or any other decisions that at least half of the members (rounded down) present decide require a vote.
+
+Decisions are made in meetings when a quorum of the members are present and may pass with at least half the members (rounded up) of the committee supporting it.
+
+Quorum is considered reached when at least half of the members (rounded up) are present.  
+Members of KSC may abstain from a vote. Abstaining members will only be considered as contributing to quorum, in the event that a vote is called in a meeting.
+
+## Getting in touch
+
+There are two ways to raise issues to the steering committee for decision:
```

Comment:
We should clarify that only KSC members can formally table an issue for a vote (as outlined in the section above).

But I agree that we should suggest where people can raise things to the attention of the KSC.